### ISSUE TITLE
Accessibility (Hard) - Focus when using keyboard

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#152]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/152) by Hailey Barnett (Bitbucket: [barnetth](https://bitbucket.org/barnetth), GitHub: [barnetth](https://github.com/barnetth)).**

----------------------------------------

I have noticed that the page does not follow keyboard movements. This means when tabbing through pages, I can reach a component that I won’t actually see due to the inability to scroll. There should be a focus on the keyboard movements so as the user tabs through the page, the user can see where they are.

This is not a specific WCAG guideline, but something that definitely needs to be fixed. If you have any questions, please let Hailey know.

**Comment by bryantal** [comment]
**Original comment by William Romano (Bitbucket: [Will Romano](https://bitbucket.org/Will Romano), ).**

----------------------------------------

The sticky footer was successfully implemented here but the changes will be merged in a PR for issue #162.

**Comment by bryantal** [comment]
**Original comment by William Romano (Bitbucket: [Will Romano](https://bitbucket.org/Will Romano), ).**

----------------------------------------

Changes merged with issue #162.
